users.go

```diff
@@ -234,6 +235,7 @@ func (u *UsersService) getuserwhere(where, username, logmsg string, opts Option)
 
 	defer resp.Body.Close()
 	PrintHeader(resp)
+	//SaveResponse(resp.Body, "test_data/upvoted.json")
```

Comment:
Should remove this comment

Reply:
This doesn't affect `Upvoted` method as this suggested change is in `getuserwhere` method.